test/example/relooper-fuzz2.c

```diff
@@ -23,7 +23,8 @@ int main() {
                    0,
                    0,
                    BinaryenTypeInt32(),
-                   BinaryenConst(module, BinaryenLiteralInt32(4))),
+                   BinaryenConst(module, BinaryenLiteralInt32(4)),
+                   "0"),
```

Comment:
For the JS API, null is appended for the memory name and this is handled inside the different calls (BinaryenLoad, BinaryenStore, etc.). For C calls, we can set a default parameter of null for the memName to avoid updating these tests.

Reply:
I see, right... a null is the best we could do, since we must still change the C API to add the new param either way. Sounds ok to leave null for later then.